script/download_rust_deps.py

```diff
@@ -184,7 +184,7 @@ def main():
         rustup_add_target('aarch64-apple-darwin')
 
     cxx_path = os.path.abspath(
-        os.path.join(BRAVE_CORE_ROOT, 'third_party', 'rust', 'cxx', 'v1'))
+        os.path.join(BRAVE_CORE_ROOT, '..', 'third_party', 'rust', 'cxx', 'v1'))
```

Comment:
It would be more clear here to use CHROMIUM_ROOT directly. See e.g. https://github.com/brave/brave-core/commit/d139b348a56f9b8571247688ea7b111a404fe4db